packages/views/__init__.py

```diff
@@ -92,9 +92,11 @@ def update(request):
         # [community] -> [extra] moves
         for pkg in Package.objects.filter(id__in=ids):
             if request.user in pkg.maintainers:
-                rels = PackageRelation.objects.filter(pkgbase=pkg.pkgbase,
-                        user=request.user,
-                        type=PackageRelation.MAINTAINER)
+                rels = PackageRelation.objects.filter(
+                           pkgbase=pkg.pkgbase,
+                           user=request.user,
+                           type=PackageRelation.MAINTAINER
+                       )
```

Comment:
This is a bit too much change because of change.

Reply:
true but it was more readable so I left it in this PR :) I will remove it